README.md

```diff
@@ -1 +1,69 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] The Pull Request should have a summary describing introduced changes in more detail, kindly add more descriptive changes to your description so it is descriptive and the user can know what changes you implemented in it

Reply:
Would you give me some directions where to write the summary in Pull Request?
Many thanks,
Amare